core/src/test/java/org/apache/accumulo/core/client/ClientConfigurationTest.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.accumulo.core.client.ClientConfiguration.ClientProperty;
 import org.junit.Test;
 
+@SuppressWarnings("deprecated")
```

Comment:
ClientConfiguration is deprecated. Rather than suppress warnings... just make this class also deprecated. It will go away when ClientConfiguration does.

Reply:
fixed in d8ccf825b2a377